src/components/DrivePicker.tsx

```diff
@@ -1,97 +1,42 @@
 import { Button, Grid } from '@material-ui/core';
 import StorageIcon from '@material-ui/icons/Storage';
-import React, { useContext, useState } from 'react';
+import React, { useContext } from 'react';
 import GooglePicker from 'react-google-picker';
 import { SpreadsheetCtx } from 'src/contexts/spreadsheet.context';
-import { UserCtx } from 'src/contexts/user.context';
-import { IUser } from 'src/types';
-import { ISpreadsheet } from 'src/types/spreadsheet';
-
+import { StepCtx } from 'src/contexts/step.context';
```

Comment:
I'm not sure about such imports, this `src/` smells

Reply:
It is default react root directory “Absolute Imports with Create React App” by Kyle Truong https://link.medium.com/witlKyPZKU

Reply:
ok